hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-resourcemanager/src/test/java/org/apache/hadoop/yarn/server/resourcemanager/recovery/TestZKRMStateStore.java

```diff
@@ -1567,4 +1568,16 @@ public void testAppSubmissionContextIsPrunedInFinalApplicationState()
         Collections.emptyMap(), ctx.getApplicationSchedulingPropertiesMap());
     store.close();
   }
+
+  @Test
```

Comment:
Not familiar with YARN code. The production code looks fine to me. The test code should add more coverage for each of the methods in ZKRMStateStoreOpDurations.

Reply:
done.  add more test in testMetricsInited to cover each of the methods in ZKRMStateStoreOpDurations.